packages/travix-ui-kit/components/stepper/stepper.scss

```diff
@@ -0,0 +1,100 @@
+.ui-stepper {
+  display: flex;
+  height: 39px;
+  justify-content: space-between;
+
+  .ui-stepper__button {
+    background: linear-gradient(to bottom,  var(--tx-generic-color-accent-top) 0%, var(--tx-generic-color-accent-bottom) 100%); /* W3C */
+    background: var(--tx-generic-color-accent-bottom); /* Old browsers */
+    border: none;
+    border-bottom: 3px solid var(--tx-generic-color-accent-border);
+    border-radius: 5px;
+    box-shadow: none;
+    color: var(--tx-generic-color-primary-lighter);
+    font-size: 22px;
+    height: 39px;
+    line-height: 39px;
+    margin:0;
+    outline: none;
+    padding: 0;
+    position: relative;
+    text-decoration: none;
+    transition: all .1s ease-in-out;
+    vertical-align: middle;
+    white-space: nowrap;
+    width: 36px;
+    z-index: 2;
+    cursor: pointer;
+
+    &:hover {
+      border-bottom: solid 3px var(--tx-generic-color-accent-border);
+      box-shadow: none;
+      color: var(--tx-generic-color-active);
+      margin: 0;
+      transition: color .1s ease-in-out;
+    }
+
+    &:active {
+      border-bottom: solid 1px var(--tx-generic-color-accent-border);
+      box-shadow: none;
+      color: var(--tx-generic-color-active);
+    }
+
+    &:before {
+      font-size: 20px;
+      width: 36px;
+    }
+
+    &-disabled {
```

Comment:
I would avoid this kind of class name concatenation, as they make project-wide searches more difficult (pseudo-classes are fine).